moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -238,6 +238,9 @@ export const ProjectsListViewQueryConf = {
     added_by: {
       type: "string",
     },
+    interim_project_id: {
+      type: "string",
```

Comment:
This works just like it is, but I noticed that `project_id` is another field that is set to type = number in `ProjectsListViewFiltersConf.js` but is set to `Int` in this file. But, I think this is a Grid Table vs. Material Table config thing so it shouldn't matter.